electron/extraction.js

```diff
@@ -23,6 +23,10 @@ async function runExtraction(fromDate, toDate, configFilepath, runLogFilepath, d
       defaultDebug,
       allEntries,
     ).then((value) => value);
```

Comment:
DP1 - Unrelated to this pr strictly speaking, but do we need this `.then`? If not, could we remove it?

Reply:
I removed the .then call.